src/main/java/training/Elbow/Elbow.java

```diff
@@ -0,0 +1,96 @@
+package training.Elbow;
+
+import com.revrobotics.CANSparkBase;
+import com.revrobotics.CANSparkLowLevel;
+import com.revrobotics.CANSparkMax;
+import edu.wpi.first.math.geometry.Rotation2d;
+import utils.GBSubsystem;
+
+import static training.Elbow.ElbowConstants.ELBOW_GEAR_RATIO;
+import static training.Elbow.ElbowConstants.ELBOW_LOG_PATH;
+import static training.Elbow.ElbowConstants.ELBOW_PID_CONTROLLER;
+import static training.Elbow.ElbowConstants.ELBOW_ID;
+import static training.Elbow.ElbowConstants.DEFAULT_POSITION_ELBOW;
+import static training.Elbow.ElbowConstants.POWER_LIMIT_ELBOW;
+import static training.Elbow.ElbowConstants.ELBOW_TOLERANCE;
+import static training.Elbow.ElbowConstants.ELBOW_FEEDFORWARD;
+import static training.Elbow.ElbowConstants.PID_SLOT;
```

Comment:
```suggestion
import static training.Elbow.ElbowConstants.*;
```
.* means everything in it

Reply:
I prefer to not import at all and do ElbowConstants.CONSTANT in code.
Also, using .* makes the performance a very, very little bit slower, so I don't like using * even if the change is very minimal.